### ISSUE TITLE
Menu Selection Should Reload

### ISSUE THREAD

**Issue by tsawyer** [AllStarLink/Allmon3#163]
When on a menu item's page, selecting the menu item again should reload the page. As it is now it does nothing. This behavior is particularly unfriendly on small screens as you are left with only the menu if you select the same item. That's because the menu takes up the full screen on small devices.

**Comment by jxmx** [comment]
@tsawyer Is your interest in just having the menu disappear if you happen to click on the same node, or to actually force a page reload? There's no technical action required of a browser when you click on its same node entry in the menu so it doesn't do anything.

**Comment by tsawyer** [comment]
It seems unusual and unexpected for a page (any page) to not load when selecting it's menu item. But however you choose to address the problem is up to you.

**Comment by jxmx** [comment]
@tsawyer I'll force the page to reload. Since this the link points back to itself already in the form of index.html#node there isn't really the expectation for the page to reload syntactically. However I get your point of user expectations.

**Comment by tsawyer** [comment]
All good except it doesn't work on the home button.

web.ini clip:
```
; Link the "Home" button will open. This is useful
; for filtering back down to an intial list of nodes
; rather than "all nodes"
HOME_BUTTON_URL = "index.html#2530,27439,28020"
```

**Comment by tsawyer** [comment]
Voters alternately return a API error... 1st click ie ok, second click errors, 3 click is ok... repeats.

<img width="1383" alt="Screenshot 2023-09-05 at 10 11 29 AM" src="https://github.com/AllStarLink/Allmon3/assets/1526362/6ebe945d-85ca-40e9-a156-e0cced3e7439">

**Comment by jxmx** [comment]
Yeah. This is a bit ugly because I'm trying to subvert the expected browser behavior of not reloading a page + anchors when you're already at that location. I'll have to think about this one some more.

**Comment by jxmx** [comment]
c4ed732

**Comment by jxmx** [comment]
Finally fixed in c4ed732..7abd497
